offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -453,19 +452,17 @@ class OfferToPDFConverter implements OfferExporter {
                         dataAnalysisItemsExists = false
                         htmlContent.getElementById("${key.toString()}-sub-overhead").remove()
                         break
+                    case ProductGroups.DATA_MANAGEMENT:
+                        dataManagementItemsExists = false
+                        htmlContent.getElementById("${key.toString()}-sub-overhead").remove()
+                        break
```

Comment:
what if it is PROJECT_MANAGEMENT ?

Reply:
The ProductGroups Enum is defined solely in the `OfferToPDFConverter` class and ProductGroups.DATA_MANAGEMENT currently stores all items associated with `DataStorage` and `ProjectManagement` products.

Reply:
Can you make it more explicit please? Like PROJECT_AND_DATA_MANAGEMENT?

Reply:
Sure will do!